product_pricelist_brand_required/models/product_pricelist.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2025 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class ProductPricelist(models.Model):
+    _inherit = "product.pricelist"
+
+    def _get_applicable_rules_domain(self, products, date, **kwargs):
+        return super()._get_applicable_rules_domain(products, date, **kwargs) + [
```

Comment:
Could you perform the domain merge with the Odoo library?

Reply:
Yes, but i think it is not needed

Reply:
@ValentinVinagre The thumb up means i have to change it or not?

Reply:
@ValentinVinagre 

Honestly, it looks much worse with the Odoo library, can we resolve al points?

<img width="798" height="170" alt="image" src="https://github.com/user-attachments/assets/2c85e871-2fe2-46c6-bfbf-5d4505e74a62" />